mautic/mautic
# [UX] Change "Import" link on Dashboard dropdown to “Import or select pre-defined dashboard”

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [X]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



### Description
The "Import" link on the Dashboard dropdown is currently not fully representative of its function.

Having easy access to templates is essential for engaging new users. Templates simplify the configuration process and serve as excellent examples of what information can be used to create a functional and interactive dashboard. They provide a starting point that users can customize to suit their specific needs, reducing the learning curve and encouraging exploration.

Currently, the problem is that these templates are somewhat hidden. Users only discover them when they go to import a dashboard and realize that there are predefined models available. This lack of visibility could be hindering user engagement and utilization of these resources.


### Impact
Addressing this issue would improve the user experience by providing clearer navigation. It would reduce potential confusion and make the platform more intuitive, particularly for new users or those less familiar with the system. This change could also potentially increase user engagement with the template selection and import features.

It would not only make the platform more user-friendly but also showcase the platform's capabilities, inspiring users to fully leverage the tools at their disposal.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. On the dashboard page, click on the actions dropdown
3. See the new link label, perhaps click on them to know the templates if you dnk yet

![image](https://github.com/mautic/mautic/assets/116097999/e4832aed-edfe-42fd-a33e-697874c2bbfe)
